### ISSUE TITLE
Don't complain about abstract attributes if class has an Any base class

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#4229]
Mypy complains about abstract method `C.f` even though `A` may well implement it:

```py
from typing import Any
from abc import abstractmethod, ABCMeta

A: Any

class D(metaclass=ABCMeta):
    @abstractmethod
    def f(self) -> None: pass

class C(A, D):
    pass

C() # Cannot instantiate abstract class 'C' with abstract attribute 'f'
```

If a class has an `Any` base class, mypy shouldn't complain about any abstract attributes.
